features/netsocket/NetworkStack.cpp

```diff
@@ -100,25 +90,13 @@ class NetworkStackWrapper : public NetworkStack
         return address->get_ip_address();
     }
 
-    virtual int gethostbyname(const char *name, SocketAddress *address)
```

Comment:
This breaks removal gethostbyname(literal, UNSPEC) - gethostbyname(version = AF_UNSPEC) doesn't actually work. Need to extend that too.